### DISCUSSION TITLE
custom names for town and nation and adding an extra

### DISCUSSION THREAD

**Discussion by Daddy-Hunger** [TownyAdvanced/Towny#7578] [Q&A] [answered]
Hey guys - I am wanting to setup a system that is the same as what already exists - but renamed and has an extra level.

The current default setup is town > nation
I wanna be able to setup guild > kingdom > Empire

is this possible?
if so, how?

**Comment by LlmDl** [comment, marked as answer]
This is not possible with Towny.
